airbyte-cdk/python/airbyte_cdk/base_python/cdk/abstract_source.py

```diff
@@ -166,9 +166,8 @@ def _read_incremental(
             yield self._checkpoint_state(stream_name, stream_state, connector_state, logger)
 
     def _read_full_refresh(self, stream_instance: Stream, configured_stream: ConfiguredAirbyteStream) -> Iterator[AirbyteMessage]:
-        args = {"sync_mode": SyncMode.full_refresh, "cursor_field": configured_stream.cursor_field}
-        for slices in stream_instance.stream_slices(**args):
-            for record in stream_instance.read_records(stream_slice=slices, **args):
+        for slices in stream_instance.stream_slices(sync_mode=SyncMode.full_refresh, cursor_field=configured_stream.cursor_field):
```

Comment:
why make this change?

Reply:
matter of taste, but I find it is more readable like this:
```python
records = stream_instance.read_records(stream_slice=slices, sync_mode=SyncMode.full_refresh, cursor_field=configured_stream.cursor_field)
for record in records:
```

Reply:
```
➜  python git:(davinchia/cdk-mypy) ✗ mypy airbyte_cdk/base_python
airbyte_cdk/base_python/cdk/abstract_source.py:170: error: Argument 1 to "stream_slices" of "Stream" has incompatible type "**Dict[str, Union[SyncMode, List[str], None]]"; expected "SyncMode"
airbyte_cdk/base_python/cdk/abstract_source.py:170: error: Argument 1 to "stream_slices" of "Stream" has incompatible type "**Dict[str, Union[SyncMode, List[str], None]]"; expected "Optional[List[str]]"
airbyte_cdk/base_python/cdk/abstract_source.py:170: error: Argument 1 to "stream_slices" of "Stream" has incompatible type "**Dict[str, Union[SyncMode, List[str], None]]"; expected "Optional[Mapping[str, Any]]"
Found 3 errors in 1 file (checked 22 source files)
```
MyPy doesn't like kwargs, since it can't be sure that the entries actually exists and are bound with the right type.

Reply:
seems reasonable to make it here